### DISCUSSION TITLE
Custom Cube with Different Packs

### DISCUSSION THREAD

**Discussion by blotsy** [Senryoku/Draftmancer#691] [Ideas]
I would love to be able to make a Cube that has different packs. The first pack drafted would be one list, defined by the text file. The second pack from a completely different list, in the same file. Basically make your own "chaos draft", kinda.

**Comment by Senryoku** [comment]
Hello,

The `layouts` and `predeterminedLayouts` settings should allow you to do just that.
Here's a simplified example:
```js
[Settings]
{
  "layouts": {
    "Cube_1": {
      "weight": 1,
      "slots": {
        "List_1": 15
      }
    },
    "Cube_2": {
      "weight": 1,
      "slots": {
        "List_2": 15
      }
    }
  },
  "predeterminedLayouts": ["Cube_1", "Cube_2"],
}
[List_1]
..Card List..
[List_2]
..Card List..
```

Basically, each layout can use completely different slots and disjoint card lists, defining completely separate cubes, or sets. `predeterminedLayouts` allows you to specify how each pack will be generated, rather than picking a random layout for each of them.


In case you missed it, the complete documentation is available here: https://draftmancer.com/cubeformat.html

**Comment by blotsy** [comment]
I'm a fool and didn't read the documentation properly / misinterpreted what
I was reading.

Thank you!
